frontend/src/scenes/early-access-features/EarlyAccessFeature.tsx

```diff
@@ -259,33 +247,12 @@ function PersonList({ earlyAccessFeature }: PersonListProps): JSX.Element {
 
     const { featureFlag } = useValues(featureFlagLogic({ id: earlyAccessFeature.feature_flag.id || 'link' }))
 
-    const optUserOut = async (person: PersonType): Promise<void> => {
```

Comment:
Removing this manual removal functionality. As discovered in testing, if a manual change happens in app, the client cache may have a stale value and continue passing as an override prop. This means the feature flag can be enabled even if the person prop has been set and saved to false

Reply:
yep makes sense